api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -310,6 +314,17 @@ message MetadataFilter {
   google.protobuf.BoolValue match_if_key_not_found = 2;
 }
 
+// Filters based on access log type.
+message LogTypeFilter {
+  // Logs only records which their type is one of the types defined in this field.
+  repeated data.accesslog.v3.AccessLogType types = 1
+      [(validate.rules).repeated = {items {enum {defined_only: true}}}];
+
+  // If included and set to true, the filter will instead block all records
```

Comment:
`If this field is set to true`? `include` sounds confusing because the field is `exclude`.

Reply:
done